.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+Reviewers: GitHub @<name>s of all reviewers
+
+Linked PRs: <links to corresponding PRs, optional>
+
+## Changes
+- <changes here>
+
+## TODOs:
+- [ ] +1
+- [ ] Updated README
+- [ ] Updated CHANGELOG
+- [ ] <Other applicable TODOs>
+- [ ] <Feedback from reviewers>
+
+## How To Test
+- <necessary config changes>
+- <necessary corresponding PRs>
+- <how to access the new / changed functionality -- fixtures, URLs>
+
+## Applicable Research Resources
+- <links, optional>
```

Comment:
My one concern is that if we put one of these in each repo, then we need to change them every time the main one changes. I think we should just link to the PR template in the mobify-code-style repo, since it's public.